### DISCUSSION TITLE
Add a user to an app using api

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#5433] [Help] [answered]
Hello,

I can't find in the API doc how I can add a user to an app?

Thank you

**Comment by melohagan** [comment, marked as answer]
Hello @oswaldB 

Is this what you were looking for? https://docs.budibase.com/reference/post_users

I think you can use the roles object to specify which app you want to create/update a user for.

**Comment by oswaldB** [comment]
Oh thank you.

It is in the role element in the payload.

Thanks for your answer.
